src/main/java/com/spotinst/sdkjava/model/ElastigroupStartDeploymentOnFailure.java

```diff
@@ -0,0 +1,120 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.spotinst.sdkjava.enums.AwsElastiGroupActionTypeEnum;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ElastigroupStartDeploymentOnFailure {
```

Comment:
Rename class to ElastigroupStrategyOnFailure

Reply:
Done